src/com/itacademy/eshop/shop/ShoppingCart.java

```diff
@@ -20,10 +22,32 @@ public void printShoppingCart() {
         System.out.println("Total price: " + getTotalPrice());
     }
 
-    public String getTotalPrice() {
-        /**
-         * Cakculates total price of shopping cart
-         */
-        throw new UnsupportedOperationException("getTotalPrice() is not implemented yet");
+    public double getTotalPrice() {
+        double sum = 0;
+        for (Product product : products) {
+            sum = product.getPrice();
+        }
+        System.out.println(sum);
+        return sum;
+    }
```

Comment:
It is good to have line breaks after each method

Reply:
Formatting. Ctrl + Alt + L (windows) Option + Command + L (macOS)